webrtc.html

```diff
@@ -7197,6 +7197,10 @@ <h2>Methods</h2>
       in order of ascending RTP timestamps. The user agent MUST keep information
       from RTP packets delivered to the <code><a>RTCRtpReceiver</a></code>'s
       <code><a>MediaStreamTrack</a></code> in the previous 10 seconds.</p>
+      <div class="note">Even if the <a>MediaStreamTrack</a> is not attached to
+      any sink for playout, <code>getSynchronizationSources</code> and
+      <code>getContributingSources</code> returns up-to-date information; sinks
```

Comment:
Would it make sense to change "returns" to "MUST return"?

Reply:
This is a note, I think we're covered by the previous "MUST" above that this note is referring to :)